### PR TITLE
edit admin account info

### PR DESCRIPTION
**Pull request by njgheorghita**

### CODE REVIEW

**Comment by Dpalazzari** [comment]
Worked on with @njgheorghita I approve. 👍

File: spec/features/admin/admin_modify_account_data_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      fill_in "user_email", with: "chicken@salad"
+      fill_in "user_password", with: "Chicken Salad"
+      click_on "Update User"
+    end
+
+    expect(current_path).to eq(admin_dashboard_path)
+    expect(page).to have_content("Chicken Salad")
+    expect(page).not_to have_content("Drew")
+  end
+end
```
Comment (Laszlo-JFLMTCO) [inline spec/features/admin/admin_modify_account_data_spec.rb:55]: @njgheorghita @Dpalazzari Good job with the `happy-path`. Can we please make sure to include testing for `sad-path`? Thank you. 👍  FYI @drod1000

Comment (Dpalazzari) [reply spec/features/admin/admin_modify_account_data_spec.rb:55]: @Laszlo-JFLMTCO Problem solved.

Comment (Laszlo-JFLMTCO) [reply spec/features/admin/admin_modify_account_data_spec.rb:55]: 🚀

**Pull request merged**